main.py

```diff
@@ -20,26 +18,26 @@
 import numpy as np
 import tensorflow as tf
 import config
+import tf_config
 import math
-# for saving to xml files
 from pascal_voc_writer import Writer
 
-
-def get_session():
-    config = tf.ConfigProto()
-    config.gpu_options.allow_growth = True
-    return tf.Session(config=config)
-
-
-keras.backend.tensorflow_backend.set_session(get_session())
-
-model_path = os.path.join('.', 'snapshots', 'resnet50_coco_best_v2.1.0.h5')
-
-model = models.load_model(model_path, backbone_name='resnet50')
+# make sure the file is inside semi-auto-image-annotation-tool-master
+import pathlib
+cur_path = pathlib.Path(__file__).parent.absolute()
+# cur_path = r"semi-auto-image-annotation-tool-master"
+sys.path.append(cur_path)
+os.chdir(cur_path)
```

Comment:
Have you tried it across platforms? I encountered an error here running it on Debian.

`TypeError: argument should be string, bytes or integer, not PosixPath`

Change line 27, 
From:
`cur_path = pathlib.Path(__file__).parent.absolute()`
To:
`cur_path = pathlib.Path(__file__).parent.absolute().as_posix()`

This fixes the problem.

This is running on Python3.5 which is the minimum required Python version.